src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -78,7 +79,18 @@ trait BackendCall {
    * expression `read_lines(my_file_var)` would have to call lookupFunction()("my_file_var")
    * during expression evaluation
    */
-  def lookupFunction: String => WdlValue
+  def lookupFunction: String => WdlValue = {
+    def resolveParameter(name: String) = Try(locallyQualifiedInputs.getOrElse(name, throw new Throwable(s"variable $name not found")))
```

Comment:
yeah, I get lazy with exceptions sometimes.

I don't think I need to be throwing the exception, probably a remnant